File: src/app/material-advisor/src/app/models/knowledge-check/topic/KnowledgeCheckAnswerGroup.ts
import { LanguageText } from "@shared/models/LanguageText";
import { KnowledgeCheckAnswer } from "./KnowledgeCheckAnswer";
import { GUID } from "@shared/types/GUID";

export interface KnowledgeCheckAnswerGroup {
  id: GUID;
  number: number;
  isTechnical: boolean;
  content: LanguageText[];
  answers: KnowledgeCheckAnswer[];
}

File: src/api/MaterialAdvisor/MaterialAdvisor.Application/Models/KnowledgeChecks/Topics/KnowledgeCheckAnswerGroup.cs
namespace MaterialAdvisor.Application.Models.Topics;

public class KnowledgeCheckAnswerGroup
{
    public Guid Id { get; set; }

    public byte Number { get; set; }

    public bool IsTechnical { get; set; }

    public IEnumerable<LanguageText>? Content { get; set; }

    public IEnumerable<KnowledgeCheckAnswer> Answers { get; set; } = [];
}
